lib/src/view/puzzle/dashboard_screen.dart

```diff
@@ -123,25 +124,33 @@ class PuzzleDashboardWidget extends ConsumerWidget {
         );
       },
       error: (e, s) {
+        final connectivity = ref.watch(connectivityChangesProvider);
+        final isOnline = connectivity.value?.isOnline ?? true;
+
         debugPrint(
           'SEVERE: [PuzzleDashboardWidget] could not load puzzle dashboard; $e\n$s',
         );
-        return Padding(
-          padding: Styles.bodySectionPadding,
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              Text(
-                context.l10n.puzzlePuzzleDashboard,
-                style: Styles.sectionTitle,
-              ),
-              if (e is ClientException && e.message.contains('404'))
-                Text(context.l10n.puzzleNoPuzzlesToShow)
-              else
-                const Text('Could not load dashboard.'),
-            ],
-          ),
-        );
+
+        if (isOnline) {
```

Comment:
This change doesn't make sense anymore I find, since now the dashboard is in its own screen.

I'd leave it as it was.